elasticai/creator/nn/fixed_point/conv1d/testbench.py

```diff
@@ -9,8 +9,8 @@
     module_to_package,
 )
 from elasticai.creator.nn.fixed_point.number_converter import FXPParams, NumberConverter
+from elasticai.creator.testing.simulated_layer import Testbench
```

Comment:
We should try to provide a clean top-level api for users. That is why the CI is ensuring that certain architectural boundaries are respected.
One of these rules is that public api is not deeper than `elasticai.creator.<name>`.
For important cases we could break with that rule, but i don't think this is such a case.
So either that simulated layer is important enough to be exported as public api. In that case we should probably make it importable as

```
from elasticai.creator.testing import TestBench
```
or
```
from elasticai.creator.testing import SimulatedLayerTestBench
```

Otherwise do you think it'd make sense to just move it into the tests package? This is at least what i'd suggest in case it is solely used for our internal tests.

Reply:
Side note: for vscode there is this plugin 
https://marketplace.visualstudio.com/items?itemName=Gauge.tach

that can will highlight as soon as an import violates a rule. Couldn't find anything like this for pycharm.